README.md

```diff
@@ -1,2 +1,36 @@
-# syna-start
-Clonable starter repository for a Syna theme based website
+# Syna Start
+
+This is a sample project that can be used to jump start your Syna project. It uses Syna theme and Hugo with sample files that introduce two simple pages, one for landing and one for the about page.
+
+To start your blog run the following commands:
+
+**Development**:
+```
+$ hugo server -D
+```
+
+**Production**:
+```
+$ hugo
```

Comment:
Add explanation, that this will generate the static files for your site in public/.